chain/default_syncer.go

```diff
@@ -208,6 +210,31 @@ func (syncer *DefaultSyncer) syncOne(ctx context.Context, parent, next types.Tip
 		return nil
 	}
 
+	pMinTs, err := parent.MinTimestamp()
+	if err != nil {
+		return err
+	}
+
+	tickets, err := next.Tickets()
```

Comment:
Blocking: this looks wrong. If a tipset has two blocks and each has two tickets (one null block) then `len(tickets)` below is 4 while I think the intention is 2. This should probably be using `Height` instead.

Reply:
@frrist this *will* work but only after
(1) you move this check to be on a single block and access that single block's ticket array
(2) the implementation actually catches up with the spec and includes one ticket per preceding null block.  Right now this doesn't happen and all blocks have a single ticket.

For now as @anorth suggests comparing a block's height with its parent height is thing to do.